src/test/java/com/github/lwhite1/tablesaw/TestData.java

```diff
@@ -18,12 +15,19 @@
  */
 public enum TestData {
 
-    BUSH_APPROVAL(new ColumnType[]{LOCAL_DATE, INTEGER, CATEGORY}, "data/BushApproval.csv"),
```

Comment:
please call me if you're not getting what I'm saying about Table#getRow().
thanks.

On Sun, Jun 19, 2016 at 9:56 AM, Larry White <ljw1001@gmail.com> wrote:

> ok.
>
> On Sun, Jun 19, 2016 at 9:56 AM, Spencer Marks <notifications@github.com>
> wrote:
>
>> In src/test/java/com/github/lwhite1/tablesaw/TestData.java
>> <https://github.com/lwhite1/tablesaw/pull/19#discussion_r67617417>:
>>
>> > @@ -18,12 +15,19 @@
>> >   */
>> >  public enum TestData {
>> >
>> > -    BUSH_APPROVAL(new ColumnType[]{LOCAL_DATE, INTEGER, CATEGORY}, "data/BushApproval.csv"),
>>
>> That delete is just the diffing tool being imperfect.
>> Or do you mean I am not using this test data yet? I am sure it will be
>> useful, but I wanted to start with something really simple since I needed
>> to create known good results form the sort. The bushes and Tornados are
>> still available as TestData though.
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/lwhite1/tablesaw/pull/19/files/74797acb18f2e68d88e7598f35893100b6ad7d08#r67617417>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe/ADRXgu3zRu_u-SDFwJxnh89B4ITpF-rPks5qNUp-gaJpZM4I5ChM>
>> .
>>
>
>